kensu/nuclio/confluent_kafka/__init__.py

```diff
@@ -1,27 +1,21 @@
+import logging
+
 from confluent_kafka import Producer as ConfluentProducer
 from kensu.nuclio.agent import kensu_add_kafka_output
 
+
 class Producer(ConfluentProducer):
     def produce(self, topic, value=None, *args, **kwargs):
         result = super().produce(topic, value, *args, **kwargs)
         try:
-            if not self._kensu_nuclio_context:
-                return result
             kensu_add_kafka_output(
-                context=self._kensu_nuclio_context,
                 topic=topic,
-                cluster_name=self._kensu_stored_config.get('bootstrap.servers') or 'unknown',
+                cluster_name=self._kensu_stored_bootstrap_servers or 'unknown',
```

Comment:
cluster name based on bootstraps? It will be inconsistent no?
cuz a topic in kafka is iid from the boostrap (and even brokers), it is a concept declared at the kafka cluster definition level.
We should request the user to provide the kafka cluster name or default on just `kafka-cluster` instead of `unknown`
my2c

Reply:
currently `cluster_name` is post-processed via LDS name (`category=logical:postprocessed-cluster-name`) mapping! but we might come up with something smarter

yes, that environment variable might work, something like this:
- default_kafka_cluster_name
- read_default_kafka_cluster_name
- write_default_kafka_cluster_name

but if we could define good enough generic rules based on bootstrap servers jobs would need less manual configuration changes!  see https://github.com/kensuio-oss/kensu-py/blob/f/nuclio-thread-local/docs/nuclio/ingest_raw_events.py#L16-L28 as example